subsys/bluetooth/controller/Kconfig

```diff
@@ -6,19 +6,24 @@
 
 menu "Nordic BLE controller"
 
-config BLECTRL_MASTER_COUNT
-	int "Number of concurrent master roles"
-	default 1
-	help
-	  Number of concurrent master roles defines how many simultaneous
-	  connections can be created with the device working as a master.
-
 config BLECTRL_SLAVE_COUNT
 	int "Number of concurrent slave roles"
-	default 1
+	default BT_MAX_CONN if (BT_PERIPHERAL && !BT_CENTRAL)
+	default 1 if BT_PERIPHERAL
+	default 0
+	range 0 BT_MAX_CONN
 	help
 	  Number of concurrent slave roles defines how many simultaneous
 	  connections can be created with the device working as a slave.
+	  NOTE: the number of master roles is defined as
+	  BT_MAX_CONN - BLECTRL_SLAVE_COUNT
+
+config BLECTRL_ERROR_ON_CONN_MISMATCH
```

Comment:
Where is this check implemented in the code?

Reply:
This should have been removed, I had another check that was removed yesterday :+1: